x/fungible/keeper/zrc20_lock_token.go

```diff
@@ -0,0 +1,82 @@
+package keeper
```

Comment:
We should have the maximum possible unit test coverage for logic added in `keeper/fungible`

Reply:
It's in the works, as this is a big PR with many moving parts it's interesting that the reviews start asap.

Reply:
Let's keep the PR in Draft mode or create issue to track unit test implementation in this case